### ISSUE TITLE
find_node() does not return instanced scenes

### ISSUE THREAD

**Issue by teedoubleuu** [godotengine/godot#10595]
**Operating system or device, Godot version, GPU Model and driver (if graphics related):**
Godot 3.0 alpha 1, Windows 10

**Issue description:**
find_node() does not see nodes that are instanced scenes. Its very annoying because I would often create a bunch of nodes in the main scene, reference them using find_node() and then convert that branch into its own scene at which point the find_node() stops working even though the name of the node stays the same and its still in the main scene as previously.

**Steps to reproduce:**
Create a node in a scene and then convert it to a scene without changing the name. The instanced scene will be invisible to find_node()

**Comment by volzhs** [comment]
did you mean that "convert it to a scene" is "Save branch as scene" in editor with RMB?

**Comment by teedoubleuu** [comment]
@volzhs Yes.

**Comment by novemberist** [comment]
I can not reproduce this on the lastest builds. find_node() seems so work fine for instanced scenes(that were saved from a branch). What does get_name() return when called on one of your instanced nodes?

**Comment by novemberist** [comment]
Here is an example project adressing both scenarios (both scenes were created thorugh "save branch as scene")
[findNode.tar.gz](https://github.com/godotengine/godot/files/1248718/findNode.tar.gz)

**Comment by teedoubleuu** [comment]
find_node() does not find scene instances added to a scene in the editor either.

**Comment by volzhs** [comment]
@teedoubleuu did you add 2 or more scene instances? and tried to find it with the same name?

**Comment by teedoubleuu** [comment]
No. I added only one instance in the editor and it has the exact name I am looking for.

**Comment by volzhs** [comment]
can you make a sample project to reproduce it?

**Comment by teedoubleuu** [comment]
Ok, I think I figured out the issue. I was using get_tree().get_root().find_node() and it did not work because root of the tree is not an actual node in the scene, but the viewport. Using find_node() in the context of viewport does nothing UNLESS you change the default argument of owner=true in find_node() to false.

This is confusing because the find_node() method promises that it will find a node with a given name, but fails to communicate that it will find it ONLY if its owned by the node using this function.

**Comment by novemberist** [comment]
Yes, i just found out the same thing. Setting owner to false also finds nodes added through scripts now, like in my example

**Comment by akien-mga** [comment]
Sounds like a documentation issue?

**Comment by SolarLune** [comment]
Hello. Yeah, it does seem like a doc issue. I stumbled upon this bug before, and once again just now. It probably would be best to document just exactly what the `owned` argument does.
